src/App.js

```diff
@@ -1,14 +1,28 @@
-/* eslint-disable */
+import React from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 import Calculator from './components/Calculator';
+import Navbar from './components/Navbar';
+import Home from './components/Home';
+import Quote from './components/Quote';
 import './index.css';
-import React, { Component } from 'react';
 
-export default class App extends Component {
-  render() {
-    return (
-      <div className="App">
-        <Calculator />
+export default function App() {
+  return (
+    <Router>
+      <div>
+        <Navbar />
+        <Switch>
+          <Route exact path="/">
+            <Home />
+          </Route>
+          <Route path="/Quote">
+            <Quote />
+          </Route>
+          <Route path="/Calculator">
+            <Calculator />
```

Comment:
## [OPTIONAL]
* I strongly suggest that you fix the positioning of the calculator component some part are not visible to users 🙏 
![image](https://user-images.githubusercontent.com/52497246/129683798-df508642-1a2b-4557-89cd-867b01a8399a.png)